### DISCUSSION TITLE
如何设置某个局域网IP全局走指定节点

### DISCUSSION THREAD

**Discussion by QCfan** [immortalwrt/homeproxy#229] [Q&A]
想实现的目标：
局域网内其他设备正常分流，指定设备IP 192.168.1.100的流量，全局走某个节点b

当前配置如下：

路由模式：自定义路由
路由端口：80,443
代理模式：TUN TCP/UDP
绕过中国流量：开启
覆盖目标地址：开启
默认出站：a_node

路由节点：
a_node：默认出站
b_node：为某个设备（例如192.168.1.100）想指定的节点

路由规则内仅有一个规则：test，内容是
卷标：test
出站：b_node
源IP字段：源 IP CIDR：192.168.1.100
私有源IP：开启

如上配置后，并未按照预想目标进行流量分流。而是局域网内所有的流量全部指定到了b_node。而不仅是192.168.1.100。
请问上述配置需要怎样修改？

**Comment by GoingTime** [comment]
同问

**Comment by GoingTime** [comment]
> 想实现的目标： 局域网内其他设备正常分流，指定设备IP 192.168.1.100的流量，全局走某个节点b
> 
> 当前配置如下：
> 
> 路由模式：自定义路由 路由端口：80,443 代理模式：TUN TCP/UDP 绕过中国流量：开启 覆盖目标地址：开启 默认出站：a_node
> 
> 路由节点： a_node：默认出站 b_node：为某个设备（例如192.168.1.100）想指定的节点
> 
> 路由规则内仅有一个规则：test，内容是 卷标：test 出站：b_node 源IP字段：源 IP CIDR：192.168.1.100 私有源IP：开启
> 
> 如上配置后，并未按照预想目标进行流量分流。而是局域网内所有的流量全部指定到了b_node。而不仅是192.168.1.100。 请问上述配置需要怎样修改？

问题已解决。私有源 IP 和源 IP CIDR 这两个选项是“或”的关系，所以你选上私有源IP后整个局域网内所有的IP都匹配上了。

另外这个源 IP CIDR 选项并没有完全按照 CIDR 的语法来。如果真的按照 CIDR 语法后面加上个 /24 之类的后缀，配置文件本身不会有任何报错，但整个匹配规则都会瘫痪，日志里会有大量的 IO 超时和连接重置的报错。
